config/prod/recover-s3-bucket.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.8/templates/S3/synapse-external-bucket.j2"
+stack_name: "recover-s3-bucket"
+stack_tags:
+  Department: "SysBio"
+  Project: "recover"
+  OwnerEmail: 'meghasyam@sagebase.org'
+  CostCenter: "RECOVER / 122500"
+parameters:
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:iam::111111111111:role/EES3-RK-AAAAAA-Recover' # Care Evolution access
```

Comment:
this does not look like a valid ARN.  you need a valid ARN otherwise AWS will fail to provision this bucket.  If you don't have a valid ARN yet you should remove this for now and then update this PR with a valid ARN at a later time.